ros_cross_compile/dependencies.py

```diff
@@ -95,3 +97,22 @@ def assert_install_rosdep_script_exists(
             'Rosdep installation script has never been created, you need to run this without '
             'skipping rosdep collection at least once.')
     return True
+
+
+class DependenciesStage(PipelineStage):
+    """Wrapper class for running the dependencies stage of the pipeline."""
+
+    def __init__(self):
+        self.name = gather_rosdeps.__name__
+
+    def __call__(self, platform: Platform, docker_client: DockerClient, ros_workspace_dir: Path,
+                 customizations: PipelineStageConfigOptions):
```

Comment:
as you assert consider adding a docstring with what type of exception this function may throw